Scripts/build-archive.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
```

Comment:
I saw that you're using "Apache License 2.0" here

Reply:
Removed license per requirements